azure-functions/acbs-function/activity-create-party/index.js

```diff
@@ -24,8 +24,8 @@ const mandatoryFields = [
 ];
 
 const createParty = async (context) => {
-  const { party } = context.bindingData;
-
+  // eslint-disable-next-line prefer-destructuring
+  const party = context.bindingData.party;
```

Comment:
Is there a reason to not destructure here? if `party` is sometimes undefined could do this and remove the eslint-disable

```js
const { bindingData } = context;

if (bindingData.party) { ..... }

```